src/components/Calculator.js

```diff
@@ -0,0 +1,33 @@
+import './style.css';
+
+function Calculator() {
```

Comment:
- [ ] according to the requirements this component should be class-based please change this before requesting another review.